pytest_splunk_addon/standard_lib/sample_generation/pytest_splunk_addon_data_parser.py

```diff
@@ -43,6 +45,7 @@ def __init__(self, addon_path: str, config_path: str):
         self._psa_data = None
         self.addon_path = addon_path
         self.match_stanzas = set()
+        self._path_to_samples = self._path_to_samples()
```

Comment:
IMO it is a bit confusing that you have the same name for a variable as for the function.

Reply:
Nice construction for troubleshooting exercise for candidates, but I do agree with @artemrys - we should avoid it in our code

Reply:
I changed the name of the method to `_get_path_to_samples` Yes it was overlooked and a bug to have the same name for both of them